projects/js-packages/publicize-components/src/components/connection-management/connection-status.tsx

```diff
@@ -26,7 +29,14 @@ export function ConnectionStatus( { connection, service }: ConnectionStatusProps
 			<span className="description">
 				{ service
 					? __( 'There is an issue with this connection.', 'jetpack' )
-					: _x( 'This platform is no longer supported.', '', 'jetpack' ) }
+					: createInterpolateElement(
+							'This platform is no longer supported. Use <link>Manual Sharing</link> instead.',
```

Comment:
A few points here:

- You missed the translation for this new string.
- Since that previous string is already translated, we reuse that and make the new one as standalone:
- Let us add a more personal touch by saying, _"You can use **our** Manual Sharing feature instead."_

```suggestion
					: createInterpolateElement(
							sprintf(
								'%1$s %2$s',
								__( 'This platform is no longer supported.', 'jetpack' ),
								__( 'You can use our <link>Manual Sharing</link> feature instead.', 'jetpack' )
							),
```

Reply:
Fixed